anomaly_detector/anomaly_detector.py

```diff
@@ -215,11 +222,28 @@ def infer(self):
         # When we reached # of inference loops, retrain models
         self.recreate_models = False
 
+    def fetch_false_positives(self):
+        """Fetch false positive from datastore and add noise to training run."""
+        _LOGGER.info("Fetching false positives from fact store")
+        try:
+            r = requests.get(url=self.config.FACT_STORE_URL + "/api/false_positive")
+            data = r.json()
+            data["feedback"]
+            fp = []
+            for msg in data["feedback"]:
+                noise = [{"message": msg}] * self.config.FREQ_NOISE
+                fp.extend(noise)
+            _LOGGER.info("Added noise {} messages ".format(len(fp)))
+            return fp
+        except Exception as ex:
+            _LOGGER.debug("Fact Store is either down or not functioning")
+            return None
+
     def run(self, single_run=False):
         """Run the main loop."""
         start_http_server(8080)
         srun = False
-        fp = None
+        fp = self.fetch_false_positives()
         while True and srun is False:
```

Comment:
`True` is unnecessary here as it's always true:) and the second part of the condition decides anyway

Reply:
Thats correct. Thank you. Fixed it.